ui/round/src/view/button.ts

```diff
@@ -167,12 +176,18 @@ export const drawConfirm = (ctrl: RoundController): VNode =>
     fbtCancel(ctrl, ctrl.offerDraw),
   ]);
 
-export const claimThreefold = (ctrl: RoundController): VNode =>
+export const claimThreefold = (ctrl: RoundController, condition: (d: RoundData) => ButtonState): VNode =>
   h(
     'button.button.draw-yes',
     {
-      hook: util.bind('click', () => ctrl.socket.sendLoading('draw-claim')),
-      attrs: { title: ctrl.noarg('claimADraw') },
+      hook: util.bind('click', () => (condition(ctrl.data).enabled ? ctrl.socket.sendLoading('draw-claim') : () => {})),
```

Comment:
I don't think you actually want a `() => {}` here? Just undefined should work, or maybe better to just use an `if`.

Reply:
Understood, fixed.